src/hackyballs/toolbox.js

```diff
@@ -18,6 +18,7 @@ var HBToolbox = GObject.registerClass(class HBToolbox extends Toolbox {
         this.connect('reset', () => {
             this._model.reset();
         });
+        this.setBusy(false);
```

Comment:
This should probably be in the parent's constructor, I think we always want to start out not-busy.